src/pages/Recipes.js

```diff
@@ -53,14 +74,27 @@ export default function Recipes() {
         />
         <button type="submit">Search</button>
       </form>
-      <StyledRecipeSection>
-        {data.map((recipe, index) => (
-          <RecipeCard key={index} recipeData={recipe} />
-        ))}
-        <StyledSorryMessage availableData={availableData}>
-          Sorry, we couldn't find any recipes... <br /> Please try again!
-        </StyledSorryMessage>
-      </StyledRecipeSection>
+      <ErrorBoundary
+        FallbackComponent={ErrorCallback}
+        onReset={() => setRecipeData([])}
+      >
+        <StyledRecipeSection>
+          {recipeData.map((recipe, index) => (
+            <RecipeCard key={index} recipeData={recipe} />
+          ))}
+          <StyledSorryMessage availableData={availableData}>
+            Sorry, we couldn't find any recipes... <br /> Please try again!
+          </StyledSorryMessage>
```

Comment:
Why not just write something like: `!availableData && <StyledSorryMessage> …`

Reply:
I agree with @VladF146 . This is how I do it too. No need for extra styling. 😬